server.py

```diff
@@ -283,7 +297,8 @@ def main():
         tornado.web.url(r"/googleapi",googleApiHandler,name="googleapi"),
         tornado.web.url(r"/bluetooth",BluetoothHandler,name="bluetooth"),
         tornado.web.url(r"/sendlike",SendlikeHandler,name="sendlike"),
-        tornado.web.url(r"/appregister",AppRegisterHandler,name="appregister")
+        tornado.web.url(r"/appregister",AppRegisterHandler,name="appregister"),
+        tornado.web.url(r"/display",DisplayHandler,name="display"),
```

Comment:
Should this line need a comma in the end?

Reply:
Because if the comma add in the end, when the next URL add to the setting, it wouldn't need to change the previous line(add a comma).
This make the git different message cleaner

Reply:
OK, I thought this would be a grammar error so make this comment.
So it's ok to add a comma in the last element in the python list?

Reply:
Yes, the list() object accept a additional comma at the end.